LibraryWebApp/Models/BookType.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LibraryWebApp.Models
+{
+    public class BookType
```

Comment:
This is a joining table for many to many relationship. Current name is very confusing. You have models for BookType and TypeOfBook. Joining tables should have name [FirstTableName][SecondTableName] => BookBookType.